lib/dor/workflow_exception.rb

```diff
@@ -0,0 +1,4 @@
+module Dor
+  class WorkflowException < ::RuntimeError
+  end
+end
```

Comment:
Very little impact. `dor-services` is the only application I've found that tried to rescue `RestClient` errors directly, and only to support improperly set up tests.

Most applications will gain the ability to support more granular exception matching, but existing handling will continue to work as expected (since we were (improperly!) just throwing a raw `Exception` before)